lib/livebook_cli/server.ex

```diff
@@ -153,10 +156,24 @@ defmodule LivebookCLI.Server do
     |> Livebook.Utils.browser_open()
   end
 
-  defp open_from_options(base_url, _opts, [url]) do
-    base_url
-    |> Livebook.Utils.notebook_import_url(url)
-    |> Livebook.Utils.browser_open()
+  defp open_from_options(base_url, _opts, [url_or_file_or_dir]) do
+    cond do
+      File.regular?(url_or_file_or_dir) || File.dir?(url_or_file_or_dir) ->
```

Comment:
I think we should have separate routes. For directory just `/?path=...` and we use the path to set initial file. For file `/home/open?path=...`. @josevalim wdyt?

Reply:
the reason i went with this is because when i submitted yesterday a pr, jose mentioned the `?file=` pattern while the PR had only the directory part implemented, so i thought that we will use this for both.

I would also prefer seperate keys since it helps us avoid branching stuff like 
`if it is a file then this else that`

Reply:
Yes, agreed on separate routes. My suggestion is to keep this patch only for directories for now and we add files later:

```suggestion
      File.dir?(url_or_file_or_dir) ->
```

Reply:
@gpopides @jonatanklosko yes, I think we should use different routes altogether. My suggestion would be `/open?file=...` to mirror `/import?url=...`. WDYT @jonatanklosko?

Reply:
To be clear, I think `/open?file=...` can be a separate PR, we just need to remove the `handle_params` clause in `HomeLive`. We can add a separate URL later.

Reply:
@josevalim technically we designate `/import` as a public route, because livebook.dev and ex_doc know about it, this way we remember about keeping backward compatibility. Opening is gonna be used only by the CLI for now, so it's still rather internal, however `/open` is definitely more consistent, so I'm not sure.

Reply:
@jonatanklosko I would go ahead and make `/open` public because we will also invoke it from the desktop app. You could say that the desktop app is part of our internals but, for sanity purposes, I would keep it using only public server APIs too.

Reply:
Alright, sounds good!